hasura-docker/metadata/actions.graphql

```diff
@@ -254,3 +254,21 @@ type GetOplogEntryByTagsResponse {
   id: Int!
 }
 
+type DownloadEvidenceOutput {
+  id: Int!
+  filename: String!
+  friendlyName: String!
+  downloadUrl: String
+  fileBase64: String
+  contentType: String
+}
+
+input DownloadEvidenceInput {
```

Comment:
I think we can drop this altogether and always return the `downloadUrl` and `fileBase64` values. Hasura will handle providing the values to the user based on their requested return values. This also simplifies the issue with the required inputs test not passing. The required input is set to `id`, but the actual input from this is `input` with an `id` value in the dict. We can change the action to:

```
type Query {
  downloadEvidence(evidenceId: Int!): DownloadEvidenceOutput
}
```